### PR TITLE
Feat/placeholder about page

### LINKED ISSUE #12: Add routing to about and meet our team

### PR DESCRIPTION
**Pull request by Deraidos**
resolves #12

### CODE REVIEW

**Review by Deraidos — COMMENTED** [review]
WIP. should be its own component

File: src/pages/about/index.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
+import Link from "next/link";
 import AboutCard from "../components/AboutCard";
 
 
 export default function TheTeam() {
   return (
     <div className="flex flex-col items-center">
-      <h1 className="mb-8">The Team</h1>
+      <h1>The Team</h1>
+      <p className="text-center"><Link href="/">Home</Link></p>
```
Comment (Deraidos) [inline src/pages/about/index.tsx:9]: temporary. should be its own component

File: src/pages/index.tsx [hunk 2]
```diff
@@ -63,6 +64,7 @@ export default function Home() {
       </Head>
       <main>
         <h1 className="text-center">This is Home</h1>
+        <p className="text-center"><Link href="/about">About</Link></p>
```
Comment (Deraidos) [inline src/pages/index.tsx]: temporary. should be its own component

**Comment by Secretmapper** [comment]
@Deraidos This currently does not build:

> Build optimization failed: found page without a React Component as default export in
> <span>pages/components/Navbar</span>

In next, files under `/pages/` serve as actual 'pages' for the site and need default export, `Navbar` doesn't have one.

However, if you notice, it doesn't make sense for `Navbar` to be a page! So the best solution here is to move the 'components' folder to be adjacent `pages`, something like:

```
root
- src
-- pages
--- index.tsx
-- components
--- AboutCard.tsx
--- Navbar.tsx
```

**Comment by Deraidos** [comment]
> 

@Secretmapper Thank you for the input! I'm aware that this causes build problems. Should be fixed on the next commit

**Pull request merged**